charts/kube-prometheus-stack.yaml

```diff
@@ -181,11 +172,11 @@ grafana:
       nginx.ingress.kubernetes.io/rewrite-target: /$2
     path: /grafana(/|$)(.*)
     hosts:
-      - null
+      - 'null'
 
   grafana.ini:
     server:
-      root_url: '__escape__(https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana)'
+      root_url: '__escape__(https://{{ "{{ grains.metalk8s.control_plane_ip }}" }}:8443/grafana)'
```

Comment:
Why it's needed ?

Reply:
It's in the commit message https://github.com/scality/metalk8s/pull/3360/commits/959c271f9f4d09ecc1a8fc3e61c02c1d793212e8 :)